.gitignore

```diff
@@ -0,0 +1 @@
+/vendor
```

Comment:
Please add the typical names for PHPCS ruleset overloads devs can use locally to `.gitignore`:
```
.phpcs.xml
phpcs.xml
```

You may also want to add a `.gitattributes` file to `export-ignore` certain files, such as the Travis script and the `phpcs.xml.dist` file from being included in the packaged release zip/download from Packagist.